docs/developer-guide/styling-and-theming.md

```diff
@@ -0,0 +1,166 @@
+# Styling and Theming
+
+The look and feel is completely customizable either using one of the included themes, or building your own. Themes are built using [less](http://lesscss.org/).  
+You can find the default theme here: https://github.com/geosolutions-it/MapStore2/tree/master/web/client/themes/default
+
+## Theme Structure
+
+```
+.
++-- themes/
+|   +-- theme-name/
+|       +-- icons/
+|           +-- icons.eot
+|           +-- icons.svg
+|           +-- icons.ttf
+|           +-- icons.woff
+|       +-- img/
+|       +-- less/
+|           +-- common.less
+|           +-- style-module.less
+|       +-- bootstrap-theme.less
+|       +-- bootstrap-variables.less
+|       +-- icons.less
+|       +-- ms2-theme.less
+|       +-- theme.less
+|       +-- variables.less
+```
+
+`theme.less` is the entry point for all the main imports and it needs to be properly required in `buildConfig.js`.
+
+imports:
+- icons.less contains font-face declaration for glyphs, it extends the bootstrap glyphicons to use custom MapStore icons
+- bootstrap-theme.less contains all the less style for bootstrap components
+- ms2-theme.less contains all the less style for MapStore components
+- variable.less
+
+below an example of entry configuration:
+```js
+entry: assign({
+    ...other entries,
+    'themes/theme-name': path.join(__dirname, 'path-to', 'theme-name', 'theme.less')
+}, ...args),
+```
+
+MapStore uses a `themeEntries` function to automatically create the entries for default themes that can be found uder the `web/client/themes` directory.
+Default themes in [`web/client/themes`](https://github.com/geosolutions-it/MapStore2/tree/ directory are useful to have an overview of the structure described above.
+
+Note: we suggest to place the theme folder inside the assets directory for MapStore project
```

Comment:
Are you sure about this? Do you mean the source less files or the compiled css files? I agree for the css, not sure about less.